change/@fluentui-react-card-c7210dae-bb58-40a1-99ee-7334aaaa6ad7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32323&commitId=39e6fd49d94c653d824debe7f689c4a2e9e9e4b1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|12| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2231| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>